tools/bin/check-duplicated-storages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
```

Comment:
Can we add a check to make sure that all the tools used below are installed, namely `kubectl` and `jq` which are not part of the GNU core utils.

Reply:
Great point, thanks, added the kubectl check. No need for jq anymore.